components/Navbar.js

```diff
@@ -18,7 +18,7 @@ export default function Navbar() {
           <a id="nav-title" className="nav-section left" aria-label="acm home">
             {/* TODO: resolve next/image issue */}
             {/* eslint-disable-next-line @next/next/no-img-element */}
-            <img src={'/images/acm_wordmark_chapter.svg'} id="acm-logo" alt="ACM at UCLA" />
+            <img src={'/images/acm_wordmark_chapter.svg'} class="acm-logo" alt="ACM at UCLA" />
```

Comment:
is this change needed?

Reply:
I changed it in case we switch `img` to `next/image` in the future. It doesn't change anything visually.

Reply:
sounds good